docs/run-preinstalled-wda.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Run Preinstalled WebDriverAgentRunner
+---
+
+XCUITest driver can launch preinstalled WebDriverAgent directly against a real device.
+It lets you start an Appium session without the `xcodebuild` command execution to improve the session start performance.
+
+`appium:webDriverAgentUrl` let you manage everything related to the WebDriverAgent, but this functionality let Appium to manage instead.
```

Comment:
I don't understand what this sentence is trying to say. Could you please explain?

Reply:
I removed this sentence to avoid possible confusion.